crates/map/src/helpers/bitmap.cairo

```diff
@@ -0,0 +1,187 @@
+// Internal imports
+
+use origami_map::helpers::power::{TwoPower, TwoPowerTrait};
+
+#[generate_trait]
+pub impl Bitmap of BitmapTrait {
+    /// Count the number of bits set to 1 in the number
+    /// # Arguments
+    /// * `x` - The value for which to count the number of bits set to 1
+    /// # Returns
+    /// * The number of bits set to 1
+    #[inline]
+    fn popcount(x: felt252) -> u8 {
+        let mut x: u256 = x.into();
+        let mut count: u8 = 0;
+        while (x > 0) {
+            count += PrivateTrait::_popcount((x % 0x100000000).try_into().unwrap());
+            x /= 0x100000000;
+        };
+        count
+    }
+
+    /// Get the bit at the specified index
+    /// # Arguments
+    /// * `x` - The bitmap
+    /// * `index` - The index of the bit to get
+    /// # Returns
+    /// * The value of the bit at the specified index
+    #[inline]
+    fn get(x: felt252, index: u8) -> u8 {
+        let x: u256 = x.into();
+        let offset: u256 = TwoPower::pow(index);
```

Comment:
The const array is `252` long, but we don't check the index here. We may add an assert or add some power of twos until `255` is covered.
This applies to `get`, `set` and `unset`.

Reply:
Indeed, gonna add the remaining powers so we wont have an additional consuming check, nice spotted